coral-trino/src/main/java/com/linkedin/coral/trino/trino2rel/TrinoCalciteTransformerMapUtils.java

```diff
@@ -0,0 +1,86 @@
+import org.apache.calcite.sql.type.SqlReturnTypeInference;
+import org.apache.calcite.sql.validate.SqlUserDefinedFunction;
+
+import com.linkedin.coral.hive.hive2rel.functions.StaticHiveFunctionRegistry;
+
+
+public class TrinoCalciteTransformerMapUtils {
+  private static final StaticHiveFunctionRegistry HIVE_REGISTRY = new StaticHiveFunctionRegistry();
+
+  private TrinoCalciteTransformerMapUtils() {
+  }
+
+  /**
+   * Creates a mapping for Trino SqlOperator name to TrinoCalciteOperatorTransformer.
+   *
+   * @param transformerMap Map to store the result
+   * @param trinoOp Trino SQL operator
+   * @param numOperands Number of operands
+   * @param calciteOperatorName Name of Calcite Operator
+   */
+  static void createTransformerMapEntry(Map<String, TrinoCalciteOperatorTransformer> transformerMap,
+      SqlOperator trinoOp, int numOperands, String calciteOperatorName) {
+    createTransformerMapEntry(transformerMap, trinoOp, numOperands, calciteOperatorName, null, null);
+  }
+
+  /**
+   * Creates a mapping from Trino SqlOperator name to Calcite Operator with Calcite Operator name, operands transformer, and result transformers.
+   * To construct Calcite SqlOperator from Calcite Operator name, this method reuses the return type inference from trinoOp,
+   * assuming equivalence.
+   *
+   * @param transformerMap Map to store the result
+   * @param trinoOp Trino SQL operator
+   * @param numOperands Number of operands
+   * @param calciteOperatorName Name of Calcite Operator
+   * @param operandTransformer Operand transformers, null for identity transformation
+   * @param resultTransformer Result transformer, null for identity transformation
+   */
+  static void createTransformerMapEntry(Map<String, TrinoCalciteOperatorTransformer> transformerMap,
+      SqlOperator trinoOp, int numOperands, String calciteOperatorName, String operandTransformer,
+      String resultTransformer) {
+    createTransformerMapEntry(transformerMap, trinoOp, numOperands,
+        createOperator(calciteOperatorName, trinoOp.getReturnTypeInference(), trinoOp.getOperandTypeChecker()),
+        operandTransformer, resultTransformer);
+  }
+
+  /**
+   * Creates a mapping from Trino SqlOperator name to Calcite UDF with Calcite SqlOperator, operands transformer, and result transformers.
+   *
+   * @param transformerMap Map to store the result
+   * @param trinoOp Trino SQL operator
+   * @param numOperands Number of operands
+   * @param calciteSqlOperator The Calcite Sql Operator that is used as the target operator in the map
+   * @param operandTransformer Operand transformers, null for identity transformation
+   * @param resultTransformer Result transformer, null for identity transformation
+   */
+  static void createTransformerMapEntry(Map<String, TrinoCalciteOperatorTransformer> transformerMap,
+      SqlOperator trinoOp, int numOperands, SqlOperator calciteSqlOperator, String operandTransformer,
+      String resultTransformer) {
+
```

Comment:
No longer registering coral operators here, if there is a Coral function we want to translate Trino to, it should already exist or to be manually added. It doesn't make sense to just "invent" coral operators here.